spec/java/util/StringBuilder.lsl

```diff
@@ -0,0 +1,37 @@
+///#! pragma: non-synthesizable
+libsl "1.1.0";
+
+library std
+    version "11"
+    language "Java"
+    url "https://github.com/openjdk/jdk11/blob/master/src/java.base/share/classes/java/lang/StringBuilder.java";
+
+// imports
+
+import java.common;
+import java/lang/_interfaces;
+import java/io/_interfaces;
+
+
+// primary types
+
+@implements("java.io.Serializable")
+@implements("java.lang.Comparable<StringBuilder>")
```

Comment:
@dog-m If this will be deleted, method will be with another type and it willn't compile:

Original: public int compareTo(StringBuilder another)

Without '<StringBuilder>': public int compareTo(Object another)